Swift 3.0 Update

dylankbuckley opened the pull request:
Updated plugin for Swift 3.0 syntax. Updated syntax (Colors, Functions, Variables) and fixed errors resulting from the changes to NSDate, including removing custom operators previously required.

dylankbuckley committed:
Update EPCalendarCell1.swift

Updated functions with correct Swift 3.0 parameters
Updated color syntax

dylankbuckley committed:
Update EPCalendarConstants.swift

Updated color syntax for Swift 3.0

dylankbuckley committed:
Update EPCalendarHeaderView.swift

Updated NSCalendar Syntax for Swift 3.0
Updated Swift 3.0 Function Syntax

dylankbuckley committed:
Update EPCalendarPicker.swift

Updated NSDate syntax for Swift 3.0
Updated variable declarations
Updated Color syntax
Updated append() syntax
Updated function parameters

dylankbuckley committed:
Update EPExtensions.swift

Remove custom date comparison operators now handled in Swift 3.0

dylankbuckley committed:
Update EPCalendarConstants.swift

Updated CGSizeMake function

dylankbuckley committed:
Update EPCalendarPicker.swift

Added @objc
Changed public to open

dmorrow commented:
I'm still getting a couple of errors when trying to use this branch.  Have you tested in XCode 8?
CGSizeMake, dispatch_async, dispatch_get_main_queue and a few others

dylankbuckley committed:
Swift 3.0 Updated & Tested on Xcode 8

dylankbuckley commented:
@dmorrow I have just gone through and tested it on Xcode 8, run on iPhone 7 Simulator and merged the updated project settings. Let me know if any more issues.

dylankbuckley committed:
Disable Dates

When initialising the view, pass in disabledDates as well as selectedDates. Disabled dates are not selectable and are greyed out.

ipraba commented:
Closing this as already updated to swift 3.0
